docker/base/create_cluster.sh

```diff
@@ -0,0 +1,21 @@
+#! /bin/bash
+mkdir -p /nodes
+echo -n > /nodes/nodemap
+for PORT in $(seq 16379 16384); do
+  mkdir -p /nodes/$PORT
+  if [[ -e /redis.conf ]]; then
```

Comment:
Why would /redis.conf be on the filesystem? If we want to include an override - I'd argue for just using the repo.